src/Handlebars.Net.Helpers/HandlebarsHelpers.cs

```diff
@@ -40,6 +40,9 @@ public static void Register(IHandlebars handlebarsContext, Action<HandlebarsHelp
             var options = new HandlebarsHelpersOptions();
             optionsCallback(options);
 
+            // https://github.com/Handlebars-Net/Handlebars.Net#relaxedhelpernaming
+            handlebarsContext.Configuration.Compatibility.RelaxedHelperNaming = options.UseCategoryPrefix;
```

Comment:
Shouldn't you check separator as well? You do not need to enable `RelaxedHelperNaming` relaxed naming if `.` is not used.

Reply:
Also you may consider adding this behaviour to documentation as it can introduce unexpected result for end-user.